src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,18 +235,45 @@ const myFade = {
 <header transition:animate={myFade}> ... </header>
 ```
 
-## Preventing client-side navigation
+## Router control
 
-Once you have enabled client-side routing on a page by adding the `<ViewTransitions />` component, every anchor on the page that links to another page on your site will be navigated via client-side routing. There are some cases where you might want to not navigate via CSR. One example is if the link is to any non-page, such as a PDF within your `public/` folder. Or an API route that produces an image.
+The `<ViewTransitions />` router handles navigation by listening to:
+
+- Clicks on `<a>` elements.
+- Backwards and forwards navigation events.
+
+The following sections discuss how you can further control when navigation occurs within the router.
+
+### Preventing client-side navigation
+
+There are some cases where you might want to not navigate via CSR. A transition will only occur when both the old and new page has the `<ViewTransitions />` component. There may be cases where you want a nice animation coming from one page, such as going from an article listing page `/articles` to an article detail page `/articles/emperor-penguins`, but you don't want an animation when coming from somewhere else on the site.
 
 In these cases, you can opt out of client-side routing on a per-link basis using the `data-astro-reload` attribute like so:
 
 ```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+<a href="/articles/emperor-penguins" data-astro-reload>
 ```
 
 These links will be ignored by the router and a full page navigation will occur.
 
+### Trigger navigation
+
+There may be occasions when you need to trigger a navigation imperatively, such as via an event that occurs within JavaScript. The `astro:transitions/client` module provides a method for doing this. In any client script or client component you can use the module like so:
+
+```js title="form.js" "navigate(href)"
+import { navigate } from 'astro:transitions/client';
+
+// Navigate to the selected option automatically.
+document.querySelector('select').onchange = (ev) => {
+  let href = ev.target.value;
+  navigate(href);
+};
+```
+
+The `navigate` method takes these arguments:
+
+- `href` - The path to the page you wish to navigate. The href must be to within the same origin. Any external links will trigger redirection.
```

Comment:
```suggestion
The `navigate` method takes one argument:

- `href` - The new page to navigate to. This must be a relative URL to a page within your own domain. Any external links will trigger redirection.
```
I can see that more options are planned, but if there's only one, we should really say one.  :smile: 

And, the example above doesn't make it clear what the format should be because the content of the link is actually in a selected value. If someone were writing a path directly, would it be a relative URL like a normal link? (e.g. `/about/`) I think more explanation is necessary for this.

Reply:
The href does not have to be relative. It can be absolute, it can even be complete with protocol, host & post. 
And while it may be too complicated to explain here, I suppose it can even have a different origin without triggering a reload if the target page has the correct CORS headers.

Suggestion:
The `navigate` method takes one argument:
- `href` - The new page to navigate to.

If an error occurs while loading the target page, the navigate function falls silently back to a normal browser load with no view transitions. Typical errors are:
* The document cannot be loaded due to an invalid `href`
* The document is not an HTML document
* The target page does not define `<ViewTransitions/>`
* The target page and the current page do not have the same origin (protocol, host, port) and there are no other measures to support CORS (Cross-Origin Resource Sharing).

Reply:
We are going to have other options, so I'll go ahead and add them here.

Reply:
These have been added now, and the wording on `href` updated.